include/nbl/builtin/hlsl/shapes/beziers.hlsl

```diff
@@ -510,6 +524,13 @@ struct Quadratic
 template<typename float_t>
 static math::equations::Quartic<float_t> getBezierBezierIntersectionEquation(NBL_CONST_REF_ARG(QuadraticBezier<float_t>) lhs, NBL_CONST_REF_ARG(QuadraticBezier<float_t>) rhs)
 {
+#ifndef __HLSL_VERSION
+    using scalar_t = double;
+#else
+    using scalar_t = emulated_float64_t<false, true>;
+#endif
```

Comment:
I want `portable_` aliases, there should really be no extra `__HLSL_VERSION` anywhere due to shaderFloat64 **except the place where `portable_float64_t` and friends are defined